### PR TITLE
Added toString() in TopicManagementResponse for logging

### PR DESCRIPTION
**Pull request by shambhand**

### CODE REVIEW

**Comment by shambhand** [comment]
@googlebot I signed it!

File: src/main/java/com/google/firebase/messaging/TopicManagementResponse.java [hunk 1]
```diff
@@ -122,5 +122,10 @@ public int getIndex() {
     public String getReason() {
       return reason;
     }
+
+    @Override
+    public String toString() {
+      return "Error{" + "index=" + index + ", reason='" + reason + '\'' + '}';
```
Comment (hiranya911) [inline src/main/java/com/google/firebase/messaging/TopicManagementResponse.java:128]: Use `MoreObjects.toStringHelper()` here. See `FirebaseApp.toString()` for an example.

**Comment by hiranya911** [comment]
Thanks @shambhand. Can we also add a unit test for the new functionality?

**Comment by shambhand** [comment]
Thanks @hiranya911 for reviewing it. 
Let me know if the `InstanceIdClientImplTest.java` is an inappropriate place for this unit test case.
`

**Pull request merged**